src/content/en/fundamentals/media/manipulating/files.md

```diff
@@ -0,0 +1,327 @@
+strip the streams I don't want. The keyword copy means I'm moving the streams
+without changing their codecs. 
+
+A list of the
+[demuxing commands](/web/fundamentals/media/manipulating/cheatsheet#demux_split_audio_and_video)
+are provided on the cheat sheet.
+
+## Change the codec
+
+Continuing downward, we arrive at the codec. As stated earlier, a codec is _not_
+the same thing as a container. Two files of the same container type could hold
+data compressed using completely different codecs. The webm format for example
+allows audio to be encoded using either [vorbis](https://en.wikipedia.org/wiki/Vorbis)
+or [opus](https://en.wikipedia.org/wiki/Opus_(audio_format)). To change the
+codec I need ffmpeg.
+
+In the last section I demuxed the audio and video like this:
+
+    ffmpeg -i myvideo.webm -vcodec copy -an myvideo_video.webm
+    ffmpeg -i myvideo.webm -acodec copy -vn myvideo_audio.webm
+
+If I need to change the audio and video codec, I would replace the `copy` keyword
+with the name of a codec. For example, this command outputs an audio file
+encoded with the aac codec.
+
+    ffmpeg -i myvideo.mp4 -vn -c:a aac myvideo.m4a 
+
+The [cheat sheet](/web/fundamentals/media/manipulating/cheatsheet#codec)
+contains a short list of codecs used for DASH and HLS, and commands needed to
+convert to them.
+
+## File properties
+
+I've unpealed the onion. Yet if you compare what I've done to the list of stated
+goals you'll see that I'm not quite done. Anong the remaining items are bitrate
+and resolution. These properties correlate to the amount of data in a media
+file. It probably goes without saying, but I'm going to say it anyway, that you
+can always lower bitrate and resolution, but you can't increase them. (Well,
+technically you could, but the quality would be bad. Don't do it.)
+
+The first to changing bitrate and resultion is to [display the file
+characteristics](/web/fundamentals/media/manipulating/cheatsheet#display_characteristics)
+and verify that your source file has a higher bitrate or resultion than your
+desired result.
+
+### Bitrate
+
+_Bitrate_ is the maximum number of bits used to encode one second of a stream.
+The more bits used to encode a second of stream, the higher the fidelity.
+
+Unsuprisingly, bitrates for the web are low. The table below compares web
+bitrates with common home video formats. Values are given in megabits per second
+(Mbs).
+
+| Format | Bitrate |
+| ------ | ------- |
+| Blue-ray | 20Mbs |
+| DVD | 6 Mbs |
+| Desktop web video | 2 Mbs |
+| 4G mobile video | 0.7 Mbs |
+| 3G mobile video | 0.35 Mbs |
+| 2G mobile video | ?? |
+
+Which value should I use for video on my web pages? The short answer is at least
+the top three. If you're serving video someplace like India, you'll want to
+include 2G as well. For demonstration purposes, I'm going to target 3G.
+
+In Shaka Packager you set the bitrate by adding the bandwidth option to the
```

Comment:
The `bandwidth` option  in Shaka Packager does not modify the bit rate of the content. It is used to finetune the bandwidth value appears in the manifests (DASH MPDs or HLS playlists). If not provided, packager will estimate from the content.